docs/common-problems.asciidoc

```diff
@@ -96,6 +97,48 @@ and it alleviates problems that might result from sudden spikes of data.
 You can increase the queue size by overriding `queue.mem.events`.
 Be aware that increasing `queue.mem.events` can significantly affect APM Server memory usage.
 
+[float]
+[[delete-old-data]]
+==== Delete old data
+
+A late symptom of a full queue is that Elasticsearch ran out of disk space and started rejecting insertions.
+In addition to other storage optimization measures you will likely need to remove old indices to make room for new data.
```

Comment:
IMO, it's clear from the context that other optimizations are necessary. I would prune this a bit (to make it easier to scan) and simply say something like: "If this happens, you will likely need to remove old indices to make room for new data."